Remotion/Web/Core/UI/Controls/TabbedMultiView.cs

```diff
@@ -268,11 +268,13 @@ private void OnTabViewInserted (TabView view)
       tab.Target = view.ID;
       _tabStrip.Tabs.Add(tab);
 
-      if (Views.Count == 2 && Views.IndexOf(_placeHolderTabView) > 0)
+      if (Views.Count == 2)
         Views.Remove(_placeHolderTabView);
-
-      if (Views.Count == 1)
+      else if (Views.Count == 1)
```

Comment:
since you combined the IFs into IF/ELSE, both blocks need braces. Didn't we implement an analyzer for this last fall?